src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmTestBrowserCommand.cs

```diff
@@ -265,7 +265,10 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
         if (Arguments.NoQuit)
             options.LeaveBrowserRunning = true;
 
-        logger.LogInformation($"Starting {driverName} with args: {string.Join(' ', options.Arguments)}");
+        if (options is ChromeOptions chromeOptions)
+            chromeOptions.PageLoadStrategy = Arguments.PageLoadStrategy.Value;
```

Comment:
making this chrome-only feels like magic behavior to me that is hard to reason about. is there a problem with applying it everywhere? I guess you want to avoid needing to change calling code to specify a different value for chrome only?

Reply:
No, there's no problem, it's just that I was not sure what other options can be and if they have this setting while the fix requires chrome-only change.
We can add `FirefoxOptions` and `EdgeOptions` but I can't find docs on this method to exist on these two drivers, there's [chrome doc](https://www.selenium.dev/documentation/webdriver/drivers/options/#pageloadstrategy) but no similar for edge/firefox. Now when you asked I tested the method on edge/firefox and it seems not to cause any problems, so we can add the line for them as well.